src/numerical_flux/entropy_cons_numerical_flux.hpp

```diff
@@ -0,0 +1,41 @@
+#ifndef __ENTROPY_CONS_NUM_FLUX__
+#define __ENTROPY_CONS_NUM_FLUX__
+
+#include <deal.II/base/tensor.h>
+#include "physics/physics.h"
+#include "numerical_flux/convective_numerical_flux.hpp"
+
+namespace PHiLiP {
+namespace NumericalFlux {
+
+/// Entropy Conserving Numerica Flux currently only for Burgers' split-form 1D.
+template<int dim, int nstate, typename real>
+class EntropyConsNumFlux: public NumericalFluxConvective<dim, nstate, real>
```

Comment:
consider moving this into `convective_numerical_flux.hpp`

Reply:
fixed as per issue #125